mozci/push.py

```diff
@@ -1170,7 +1173,11 @@ def classify(
         if len(regressions.unknown) == 0 and len(regressions.intermittent) >= 0:
             return PushStatus.GOOD, regressions
 
-        # Fallback to unknown
+        # Fallback to unknown failures and retrigger each unknown task
+        if should_retrigger:
+            for _, tasks in regressions.unknown.items():
+                for task in tasks:
+                    task.retrigger()
```

Comment:
this most likely retriggers the task a single time, typically we will need more data, I would vote for 4 times (when added to the original task, that would equal 5 total tasks to determine status).  The problem is that if we already have a retrigger and both failed for different reasons, we would have a large number of extra tasks that are not needed.

Think about how to extend this, but for the purposes of this review and PR, lets focus on a single retrigger.

Reply:
@jmaher I can extend the code to call retrigger x4 if that is our end goal. Do we want to get that in now?

Reply:
can you make it a variable for number of retriggers, and then we can set it now to a default value of 1.

Reply:
Sounds good. Updated the PR